### PR TITLE
make golden sick patient for demo flow

### PR DESCRIPTION
**Pull request by fzhao99**
# PULL REQUEST

## Summary
Makes a very sick patient for use in the patient discovery page. Putting it up to make sure it looks semi-reasonable before progressing

## Related Issue
Part one #2645. Once merged, will post the patient to OpenHAPI and handle the autopopulation in the final ticket to align the patient discovery page with our designs

## Acceptance Criteria
Since we're removing the dropdowns from patient discovery to later stages, we'll need a golden patient that has all the relevant use case sicknesses to populate at once, which will allow us to get all the relevant use cases later on

## Additional Information
Anything else the review team should know?

### CODE REVIEW

File: containers/tefca-viewer/src/app/tests/assets/GoldenSickPatient.json [hunk 1]
```diff
@@ -0,0 +1,10893 @@
+// Patient ID: f288c654-6885-4f48-999c-48d776dc06af
```
Comment (m-goggins) [inline containers/tefca-viewer/src/app/tests/assets/GoldenSickPatient.json:1]: Probably want to delete these

File: containers/tefca-viewer/src/app/tests/assets/GoldenSickPatient.json [hunk 2]
```diff
@@ -0,0 +1,10893 @@
...
+// Practictioner ID: 8085737b-28ec-4b23-8604-83fd20076933
+
+{
+  "resourceType": "Bundle",
+  "id": "6cc4ca7f-f9bf-445f-999f-86eaa2e180ae",
+  "meta": {
+    "lastUpdated": "2024-10-04T15:52:02.656+00:00"
+  },
+  "type": "searchset",
```
Comment (m-goggins) [inline containers/tefca-viewer/src/app/tests/assets/GoldenSickPatient.json:10]: You might run into an issue loading into HAPI if you keep the type as `searchset`. If you do, I would try `transaction` instead.

File: containers/tefca-viewer/src/app/tests/assets/GoldenSickPatient.json [hunk 3]
```diff
@@ -0,0 +1,10893 @@
...
+  "total": 6,
```
Comment (m-goggins) [inline containers/tefca-viewer/src/app/tests/assets/GoldenSickPatient.json:11]: I'm not sure if the total needs to make the actual number of resources when uploading

File: containers/tefca-viewer/src/app/tests/assets/GoldenSickPatient.json [hunk 4]
```diff
@@ -0,0 +1,10893 @@
...
+            },
+            "system": "http://hospital.smarthealthit.org",
+            "value": "8692756"
+          }
+        ],
+        "active": true,
+        "name": [
+          {
+            "family": "Unlucky",
+            "given": ["Hyper", "A."],
```
Comment (m-goggins) [inline containers/tefca-viewer/src/app/tests/assets/GoldenSickPatient.json:97]: 😆

**Review by m-goggins — APPROVED** [review]
Looks good to me, though it will be easier to see if we're missing anything once the data is loaded. Left a few comments but nothing that should be a blocker.

File: containers/tefca-viewer/src/app/tests/assets/GoldenSickPatient.json (hunk 3 above)
Comment (fzhao99) [reply containers/tefca-viewer/src/app/tests/assets/GoldenSickPatient.json:11]: dan and I just chatted and he mentioned that we can remove this field

**Review by DanPaseltiner — APPROVED** [review]
LGTM. Thanks for putting this together.

**Pull request merged**